packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -401,8 +401,12 @@ export default mixins(
         on: {
           input: this.dateClick,
           'update:table-date': (value: string) => this.tableDate = value,
-          'click:date': (value: string) => this.$emit('click:date', value),
-          'dblclick:date': (value: string) => this.$emit('dblclick:date', value),
+          ...Object.keys(this.$listeners).reduce((on, eventName) => {
+            if (eventName.match(/:date$/)) {
+              Object.assign(on, { [eventName]: (value: string) => this.$emit(eventName, value) })
```

Comment:
`on[eventName] = ...`

Reply:
yup, i think i had immediate returning updated object in my mind